docs/site/HasMany-relation.md

```diff
@@ -265,6 +265,54 @@ export class Order extends Entity {
 }
 ```
 
+If you need to use another attribute other than the id property to be the source
+key, customizing `keyFrom` field would allow you to do so:
+
+```ts
+export class Customer extends Entity {
+  @property({
+    type: 'number',
+    id: true,
+  })
+  id: number;
+
+  // if you'd like to use this property as the source id
+  // of a certain relation that relates to a model `Foo`
+  @property({
+    type: 'number',
+  })
+  anotherKey: number;
+
+  @hasMany(() => Foo, {keyFrom: 'anotherKey'})
```

Comment:
nitpick: maybe use a more valid example here instead of `Foo`? Like what you have in the tests.